hack/build/multi-arch.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2023 NVIDIA CORPORATION
```

Comment:
This is not blocking, but is this intentional? I believe we normally do copy right kubevirt authors.

Reply:
> This is not blocking, but is this intentional? I believe we normally do copy right kubevirt authors.

I just copy the file from kubevirt/kubevirt project and this is the copyright in the file:
https://github.com/kubevirt/kubevirt/blob/main/hack/multi-arch.sh

Reply:
Not a huge deal. I am fine with this.